config/purposes/duplex_seq.yml

```diff
@@ -0,0 +1,63 @@
+# This configuration file is for the plate and tube purposes required for the
+# high throughput Duplex-Seq pipeline
+---
+LDS Stock:
+  :asset_type: plate
+  :stock_plate: true
+  :cherrypickable_target: false
+  :input_plate: true
+  :presenter_class: Presenters::StockPlatePresenter
+LDS Cherrypick:
+  :asset_type: plate
+  :stock_plate: true
+  :cherrypickable_target: true
+  :input_plate: true
+  :presenter_class: Presenters::StockPlatePresenter
+LDS Stock XP:
+  :asset_type: plate
```

Comment:
presume the default for all the missing ones here is false? e.g. stock_plate, cherrypickable_target...